orchestration/src/test/java/com/sap/ai/sdk/orchestration/OrchestrationUnitTest.java

```diff
@@ -258,14 +260,22 @@ void filteringLoose() throws IOException {
                     .withBodyFile("filteringLooseResponse.json")
                     .withHeader("Content-Type", "application/json")));
 
-    final var filter =
+    final var azureFilter =
         new AzureContentFilter()
             .hate(ALLOW_SAFE_LOW_MEDIUM)
             .selfHarm(ALLOW_SAFE_LOW_MEDIUM)
             .sexual(ALLOW_SAFE_LOW_MEDIUM)
             .violence(ALLOW_SAFE_LOW_MEDIUM);
 
-    client.chatCompletion(prompt, config.withInputFiltering(filter).withOutputFiltering(filter));
+    ContentFilter llamaFilter =
```

Comment:
(Minor)

final var

Reply:
`var` doesn't work, because the type can't be inferred from this lambda. But added the final now (even though we don't always do this for test code)